pantheon-bundle/frontend/src/app/bulkOperationConfirmation.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { Modal, ModalVariant, Button, Title, TitleSizes, AlertActionCloseButton, Alert, AlertActionLink, Progress, ProgressVariant, ProgressSize, List, ListItem, ProgressMeasureLocation } from '@patternfly/react-core';
+import WarningTriangleIcon from '@patternfly/react-icons/dist/js/icons/warning-triangle-icon';
+import "@app/app.css";
+
+export interface IBulkOperationProps {
+  header: string
+  subheading: string
+  updateSucceeded: string
+  updateFailed: string
+  updateIgnored: string
+  footer: string
+  progressSuccessValue: number
+  progressFailureValue: number
+  progressWarningValue: number
+  onShowBulkEditConfirmation: (showBulkEditConfirmation) => any
+  onMetadataEditError: (metadataEditError) => any
+}
+
+class BulkOperationConfirmation extends React.Component<IBulkOperationProps, any>{
+  constructor(props) {
+    super(props);
+    this.state = {
+      isModalOpen: false,
+    };
+
+  }
+
+  render() {
+    const { isModalOpen } = this.state;
+
+    const header = (
+      <React.Fragment>
+        <Title id="bulk-operation-more-details" headingLevel="h1" size={TitleSizes['2xl']}>
+          {this.props.header}
+        </Title>
+        <p className="pf-u-pt-sm">{this.props.subheading}</p>
+      </React.Fragment>
+    );
+
+    const footer = (
+      <Title headingLevel="h4" size={TitleSizes.md}>
+        <span className="pf-u-pl-sm">{this.props.footer}</span>
+      </Title>
+    );
+
+    return (
+      <React.Fragment>
+        <div className="p2-search__pf-c-alert">
+          <Alert
+            // className="p2-search__pf-c-alert pf-c-alert pf-m-info"
```

Comment:
should this Alert have a classname?

Reply:
by default the className is pf-c-alert pf-m-info. If attempted to add a new className to it called "p2-search__pf-c-aert" to fix the position issue.  But it didn't work. I had tomove the p2-search__pf-c-alert" class to an outer div.